### PR TITLE
Remove turbo

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,95 @@
...
+## Tests
 
-            {
-              "status": "ok",
-              "message": "Successfully Authenticated"
-            }
+All the tests are in the "test/" directory. The cloud app is using mocha as the test runner. 
 
-+ Response 401 (application/json)
+### Unit tests
```
Comment (rachael-oregan) [inline README.md]: Same as other PR, just add that mongod needs to be running locally and run `grunt serve` before running the tests

**Review by rachael-oregan — APPROVED** [review]
LGTM, just one small docs comment. I've verified that `npm run unit` and `npm run coverage` work

**Comment by danielpassos** [comment]
@rachael-oregan Do you wanna do a second round review before I merge?

**Comment by rachael-oregan** [comment]
@danielpassos looks good, you can merge

**Pull request merged**